src/Execution/MutationExecutor.php

```diff
@@ -77,14 +117,23 @@ protected static function saveModelWithBelongsTo(Model $model, Collection $remai
     protected static function handleHasManyCreate(Collection $multiValues, HasMany $relation)
     {
         $multiValues->each(function ($singleValues) use ($relation) {
-            self::executeCreate(
-                $relation->getModel()->newInstance(),
-                collect($singleValues),
-                $relation
-            );
+            self::executeCreate($relation->getModel()->newInstance(), collect($singleValues), $relation);
```

Comment:
Such complicated method calls should be split into multiple lines.

This helps readability and the ability to debug issues. For example, if `$relation->getModel()->newInstance()` fails, you would get an exact line number pointing to that.